src/Pyz/Service/Flysystem/FlysystemDependencyProvider.php

```diff
@@ -22,12 +22,20 @@ class FlysystemDependencyProvider extends SprykerFlysystemDependencyProvider
     protected function addFilesystemBuilderPluginCollection($container): Container
     {
         $container->set(static::PLUGIN_COLLECTION_FILESYSTEM_BUILDER, function (Container $container) {
-            return [
-                new FtpFilesystemBuilderPlugin(),
-                new LocalFilesystemBuilderPlugin(),
-            ];
+            return $this->getFilesystemBuilderPluginCollection();
```

Comment:
Do we need to keep `addFilesystemBuilderPluginCollection()` method in the project ?
'cause we have the same in the core module and now it is possible to manage plugins via `getFilesystemBuilderPluginCollection` method. 
https://github.com/spryker/flysystem/blob/70beda81e0a4b842b461e83f8e5bc06adc30b4db/src/Spryker/Service/Flysystem/FlysystemDependencyProvider.php#L62-L66